src/RageSoundManager.cpp

```diff
@@ -100,8 +100,8 @@ void RageSoundManager::Update()
 	/* Scan m_mapPreloadedSounds for sounds that are no longer loaded, and delete them. */
 	g_SoundManMutex.Lock(); /* lock for access to m_mapPreloadedSounds, owned_sounds */
 	{
-		std::map<RString, RageSoundReader_Preload *>::iterator it, next;
-		it = m_mapPreloadedSounds.begin();
+		auto it= m_mapPreloadedSounds.begin();
+		auto next= it;
```

Comment:
Wouldn't it make more sense if auto next was defined within the while loop instead?